src/components/OntologyBrowser/index.tsx

```diff
@@ -205,64 +211,72 @@ class OntologyModal extends React.Component<ModalProps, ModalState> {
     const disabledSameTerms = desactivateAllSameTerms(allSameTerms, dataSource);
 
     return (
-      <Modal
-        style={{ height: '80vh', maxWidth: 1400 }}
-        title={`${title} Browser`}
-        visible={isVisible}
-        onOk={() => this.onApply(targetKeys)}
-        okText={'Apply'}
-        okButtonProps={{ disabled: hasError }}
-        onCancel={this.onCancel}
-        cancelText="Cancel"
-        width="90%"
+      <ExtendedMappingProvider
+        projectId={arrangerProjectId}
+        graphqlField={ARRANGER_API_PARTICIPANT_INDEX_NAME}
       >
-        {hasError ? (
-          <Result
-            status="error"
-            title="An error occurred"
-            subTitle="Please cancel and try again."
-          />
-        ) : (
-          <Transfer
-            dataSource={disabledSameTerms}
-            targetKeys={targetKeys}
-            onChange={this.onChange}
-            render={(item: TransferItem): RenderResult => item.title || item.key}
-            disabled={false}
-            showSelectAll={false}
-            locale={{
-              notFoundContent: this.isLoadingOrEmpty() ? (
-                <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} />
-              ) : (
-                'Select items from the panel on the left in order to add them to your query'
-              ),
-            }}
+        {({ extendedMapping = [] }) => (
+          <Modal
+            style={{ height: '80vh', maxWidth: 1400 }}
```

Comment:
css vs style => rootClassName={'myCss'}

Reply:
this is preexisting code, I'll check what a can quickly fix

Reply:
oh I remember, there is a but in antd modal that wipe out your class name, so It can't work for now